### ISSUE TITLE
`sonarr_download_client` custom script variable does not report correct download client

### ISSUE THREAD

**Issue by agneevX** [Sonarr/Sonarr#4836]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Initially reported in #4801, the `sonarr_download_client` variable still reports `qBittorrent` instead of the name of the actual download client.

### Expected Behavior

Name of actual download client should be reported

### Environment

```markdown
- OS:
- Sonarr: 3.0.6.1432
- Docker Install:
- Using Reverse Proxy:
- Browser:
```

### What branch are you running?

Develop

### Trace Logs?

-

### Anything else?

-

**Comment by BaukeZwart** [comment]
For me the `sonarr_download_client` variable doesn't give any output at all.
tested with nightly 4.0.1.5764, docker install.

tested this by adding the following to my Radarr PP script.

```
echo "Download client: ${radarr_download_client}" > /config/logs/script.log
echo "Full path to the folder the episode file was imported from: $radarr_moviefile_sourcefolder}" >> /config/logs/script.log
```
`${radarr_download_client}` doesn't report anything
`$radarr_moviefile_sourcefolder}`does report the full path.

**Comment by markus101** [comment]
@agneevX the issue you linked is related to history, not connections.

@BaukeZwart you’re mixing Radarr and Sonarr (none of that is related to Sonarr).

**Comment by BaukeZwart** [comment]
oops, sorry.
wrong git
